velox/exec/HashTable.cpp

```diff
@@ -316,10 +386,10 @@ void HashTable<ignoreNullKeys>::groupProbe(HashLookup& lookup) {
     state3.preProbe(tags_, sizeMask_, lookup.hashes[row], row);
     row = rows[probeIndex + 3];
     state4.preProbe(tags_, sizeMask_, lookup.hashes[row], row);
-    state1.firstProbe(table_, 0);
-    state2.firstProbe(table_, 0);
-    state3.firstProbe(table_, 0);
-    state4.firstProbe(table_, 0);
+    state1.firstProbe<ProbeState::Operation::kInsert>(table_, 0);
```

Comment:
Hi @oerling , for kNormalizedKey mode, seems should pass -8 instead of 0 as firstKey because the normalized key is 8 bytes below row.